src/peer-routing/types.d.ts

```diff
@@ -0,0 +1,8 @@
+import PeerId from 'peer-id'
+import Multiaddr from 'multiaddr'
+
+declare class PeerRouting {
```

Comment:
its not exported ?

Reply:
I was doing a named import on `libp2p`, but it makes sense to export this by default. I changed it here and in the other relevant types files